db/migrate/20220914155226_create_user.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUser < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :Name
+      t.string :Photo
+      t.text :Bio
+      t.integer :PostsCounter
```

Comment:
- [ ] **Optional:** Going forward, kindly note that migration files follow and `snake_case` naming convention and not `CamelCase` as you currently have. [See more here.](https://gist.github.com/iangreenleaf/b206d09c587e8fc6399e)